CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1040,8 +1040,29 @@ local function CreateSelector(selectionStringTable, startPosition)
 			local isSelected = (i == index)
 
 			if not selectionLabel:IsDescendantOf(game) then
-				this.CurrentIndex = i
-				indexChangedEvent:Fire(index)
+				for i,selectionLabel in pairs(this.Selections) do	
```

Comment:
hmm, it seems like this code already handles this case?

`if not selectionLabel:IsDescendantOf(game) then
				this.CurrentIndex = i
				indexChangedEvent:Fire(index)
				return
			end`

If the issue is the gui does not update properly now because we exit out of doing all the ui updates. I think the cleanest solution would be to:

1) get rid of this if check and return
2) do all the normal tweening logic as it is now, but when we get to the tween calls, check if we are a descendantOf(game), if so do actual tweening calls, otherwise just set the positions/properties directly.